cheddar/cheddar-persistence/src/main/java/com/clicktravel/cheddar/infrastructure/persistence/database/query/AttributeQuery.java

```diff
@@ -34,4 +34,34 @@ public String toString() {
         return "AttributeQuery [attributeName=" + attributeName + ", condition=" + getCondition() + "]";
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
`hashCode()` and `equals()` should also consider `super`. To use the codegen correctly in Eclipse, first apply codegen on `AbstractQuery`, save the file, then apply codegen on `AttributeQuery`. You should then find the generated methods on `AbstractQuery` correctly include `super`.